packages/fuels-test-helpers/src/script.rs

```diff
@@ -15,26 +15,35 @@ use fuels_types::errors::Error;
 /// without having to setup a node or contract bindings.
 pub async fn run_compiled_script(
```

Comment:
At some point we should somehow consolidate this, `ScriptBuilder` and the `Script` abstraction into a consistent interface.

Just leaving this as note, not expecting any changes.